resources/views/dashboard/admin/store/edit.blade.php

```diff
@@ -0,0 +1,61 @@
+@extends('layouts.dashboard')
+
+@section('title')
+Edit Store Item
+@endsection
+
+@section('content')
+@include('inc.header', ['title' => 'Edit Store Item'])
+
+<div class="container">
+    {{ html()->form()->route('saveStoreItem', [$store_item->id])->attribute('enctype', 'multipart/form-data')->open() }}
+        @csrf
+        <div class="form-group">
+            <div class="row">
+                <div class="col-sm-6">
+                    <label for="apt">Title</label>
+                    {{ html()->text('title', $store_item->title)->class(['form-control'])->placeholder('Required') }}
+                </div>
+                <div class="col-sm-6">
+                    <label for="sim">Flags</label>
+                    {{ html()->select('flag', [
+                        '' => 'None',
+                        'coming soon' => 'Coming Soon',
+                        'limited stock' => 'Limited Stock',
+                        'out of stock' => 'Out of Stock',
+                        'new' => 'New'
+                    ], $store_item->flag)->class(['form-control']) }}
+                </div>
+            </div>
+        </div>
+        <div class="form-group">
+            <div class="row">
+                <div class="col-sm-12">
+                    <label for="price">Description</label>
+                    {{ html()->textarea('description', $store_item->description)->class(['form-control'])->placeholder('Required') }}
+                </div>
+            </div>
+        </div>
+        <div class="form-group">
+            <div class="row">
+                <div class="col-sm-6">
+                    <label for="price">Price</label>
+                    {{ html()->text('price', $store_item->price)->class(['form-control'])->placeholder('Digits only ex 1.99 (Required)') }}
```

Comment:
Can this be changed to use `->type('number')`? Then the user won't be able to enter letters (they currently can, even if you aren't supposed to)

Reply:
Probably related to this, but this broke when I typed "1264" into this field; maybe communicate the range of the field here as well.
![image](https://github.com/user-attachments/assets/ac2078f0-f41d-4369-9466-c81766cc1dab)

Reply:
Fixed - adjusted field to number type, added min/max guardrails, and added form validation to backend